client/components/data/media-list-data/index.jsx

```diff
@@ -26,6 +26,7 @@ module.exports = React.createClass( {
 
 	propTypes: {
 		siteId: React.PropTypes.number.isRequired,
+		source: React.PropTypes.string,
```

Comment:
Oh! Are the props in this component the ones used by the list-store query? I am not familiar with flux stores and how the query and Flux controller props play together, so I'm going to blindly trust you on this one taking into account that this is a work in progress and it doesn't break anything on my testing :)